src/legacy/core_plugins/kibana/public/discover/doc_table/components/table_row.js

```diff
@@ -93,18 +91,14 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
 
         // empty the details and rebuild it
         $detailsTr.html(detailsHtml);
-
         $detailsScope.row = $scope.row;
-        $detailsScope.uriEncodedId = encodeURIComponent($detailsScope.row._id);
+        $detailsScope.hit = $scope.row;
```

Comment:
Dear valued reviewer, this is the only real change here, everything else is a victim of auto formatting